### ISSUE TITLE
Shared webhost issue

### ISSUE THREAD

**Issue by sebyke** [soonick/poMMo#62]
Hi there,

it is a great looking simple programme!!!! But my webhost provider said the following thing:

It looks like the issue here is related to 'loopback' connections.

Having had a look at this file:

/home/sites/myweb.co.uk/public_html/pommo/inc/classes/mailctl.php

as far as I can see - that attempts to create an HTTP connection to a page on the same server.

For security and performance reasons (as these are shared servers) these kind of connections are blocked on our servers, and they're not something we can unblock I'm afraid, it's a server wide settings.

If you can set the script to use a local file path, that is

/home/sites/myweb.co.uk/public_html

as opposed to

http://myweb.co.uk/

it should work OK though.

any help with that? please... appreciate your hard work

**Comment by soonick** [comment]
This seems to be a duplicate of:
https://github.com/soonick/poMMo/issues/31
